entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,267 @@
+// @ts-nocheck
+import cn from '../../i18n/callerName.json';
+import sd from '../../i18n/speed.json';
+import ms from '../../i18n/music_info.json';
+
+export default {
+    data: {
+        //Left
+
+        title: "This is  title",
+        leftindicator:'/common/images/header/left_indicator_on.png',
+        lowbeam: '/common/images/header/low_beam.png',
+        bluetooth:"/common/images/header/bluetooth.png",
+        sidestand:"/common/images/header/side_stand.png",
+        highbeam:"/common/images/header/high_beam_on.png",
+        indicatoroff:"/common/images/header/right_indicator_off.png",
+        fuel:"/common/images/left/fuel.png",
+        level10:"/common/images/left/level_10.png",
+        level20:"/common/images/left/level_20.png",
+        level40:"/common/images/left/level_40.png",
+        level60:"/common/images/left/level_60.png",
+        level80:"/common/images/left/level_80.png",
+        level100:"/common/images/left/level_100.png",
+        callericon:"/common/images/middle/user_pic.png",
+        callaccept:"/common/images/middle/call_connect.png",
+        callreject:"/common/images/middle/call_disconnect.png",
+        music_cover:"/common/images/right/music_image.png",
+        previousplay:"/common/images/right/previous.png",
+        pause:"/common/images/right/pause.png",
+        nextplay:"/common/images/right/next_on.png",
+        menu:"/common/images/bottom/menu.png",
+        location:"/common/images/bottom/location.png",
+        home:"/common/images/bottom/home.png",
+        headset:"/common/images/bottom/Hearphone_select.png",
+        phone:"/common/images/bottom/contact_icon.png",
+
+
+        //onclick images
+        low_beamon:'/common/images/onClick/low_beam_on.png',
+        left_indicatoroff:'/common/images/onClick/left_indicator_off.png',
+        bluetooth_off:'/common/images/onClick/bluetooth_off.png',
+        side_stand_on:'/common/images/onClick/side_stand_on.png',
+        high_beam_off:'/common/images/onClick/high_beam_off.png',
+        right_indicator_on:'/common/images/onClick/right_indicator_on.png',
+        menu_select:'/common/images/onClick/menu_select.png',
+        location_select:'/common/images/onClick/location_select.png',
+        Hearphone_off:'/common/images/onClick/Hearphone.png',
+        contact_select:'/common/images/onClick/contact_select.png',
+
+
+
+
+        Front30PSI:"Front 30 PSI",
+        Rear32PSI:"Rear 32 PSI",
+        E:"E",
+        R:"R",
+        SPORT:"SPORT",
+        80:"80 %",
+        110:"110",
+        Ekm:"Est Km Range Left",
+
+        time:"01 : 20 PM",
+
+        callername:"",
+        Incomingcall:"Incoming call ... ",
```

Comment:
Instead of hardcoding, listen for any incoming call in the device and display that notification